px-data-grid.html

```diff
@@ -366,6 +403,16 @@
             this._updateActionMenu();
           });
         }
+
+        _loadingChanged(loading) {
+          clearTimeout(this._spinnerHiddenTimeout);
+
+          if (loading) {
+            this._spinnerHiddenTimeout = setTimeout(() => this._spinnerHidden = false, 500);
```

Comment:
`500` looks like a magic number here. should we provide the ability to change the debounce timeout? Something like

```js
loadingSpinnerDebounce: {
  type: Number,
  value: 500
}

Reply:
Done.